### PR TITLE
DDF-2065 - Initial Backend Changes for new UI

### PR DESCRIPTION
**Pull request by rzwiefel**
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@kcwire
@pklinef

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by jlcsmith** [comment]
Needs to be re-based please

File: catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java [hunk 1]
```diff
@@ -202,6 +202,10 @@
      *
      * @since DDF-2.9.0
      */
+    public static final String RELATED = "metacard.associations.related";
```
Comment (kcwire) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java]: Please add javadoc describing the usage of these attributes.

Comment (rzwiefel) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java]: done

**Comment by jlcsmith** [comment]
lgtm

**Comment by kcwire** [comment]
👍

**Pull request merged**